docs/howto/object-storage/s3/versioning.md

```diff
@@ -50,8 +65,8 @@ that in unversioned buckets:
 
 * If the object does not already exist, it is created (as in an
   unversioned bucket).
-* If the object does already exist, it is not replaced. Instead, a new
-  version appears in addition to the old one.
+* If the object already exists, it is not replaced.
+Instead, the new version becomes the current one.
```

Comment:
I've applied this, but this is actually wrong as the two-space indent is missing before "Instead". I'll fix this myself.